docs/UserGuide.md

```diff
@@ -1,35 +1,25 @@
-# AddressBook Level 4 - User Guide
+# TuLuist - User Guide
```

Comment:
I think it's ToLuist instead of TuLuist? Just flagging this so you and @louislai notice...